src/types/isEmail.js

```diff
@@ -0,0 +1,23 @@
+export default email => {
+
+    if (!email) return false;
+
+    let splits = email.split("@");
+
+    // if only one split return false
+    if (splits.length < 2) return false;
+
+    // if the first part is smaller than a character
+    if (splits[0].length === 0) return false;
+    if (splits[1].length === 0) return false;
+    if (splits[splits.length - 1].length === 0) return false;
+
+    splits = splits[1].split(".");
```

Comment:
@Lapixx I think then we sholdn't bother with this check
```
if (splits.length < 2) return false;
```

Because `test@.test` shouldn't be correct as well as `test@test..`. So we split and we check that the parts are not empty?